### DISCUSSION TITLE
Remove ► symbol from search results

### DISCUSSION THREAD

**Discussion by mikelerch** [fish-shell/fish-shell#10644] [Q&A]
<img width="274" alt="Screenshot 2024-07-31 at 15 44 26" src="https://github.com/user-attachments/assets/345edbf5-36db-416e-ac9d-71ad21944c01">

When performing a search, is there a way to format the output of the results? I want to get rid of the ► symbol in front of each command and just show the actual text.

Alternatively, is there a way to change this symbol?

**Comment by krobelus** [comment]
The easiest way is to modify the source.

Actually, I think we want to remove the ► symbol for everyone.
The commit that introduced it (1af9b8e43 (Prefix history pager results with a fake prompt, 2022-07-27)) seems to imply that part of the reason for adding the prefix was lack of syntax highlighting, which we now have.
